user/reference/xenial.md

```diff
@@ -0,0 +1,153 @@
+---
+
+## What This Guide Covers
+
+On this page, you can find what is included on the Xenial build environment.
+
+## Using Xenial
+
+To get your builds using Ubuntu Xenial 16.04, add the following to your `.travis.yml`:
+
+```yaml
+dist: xenial
+```
+{: data-file=".travis.yml"}
+
+Please note that Xenial is available on our fully virtualized
+infrastructure. If you are running an Enterprise installation, please reach out
+to [enterprise@travis-ci.com](mailto:entereprise@travis-ci.com) to see how you can use the Xenial Docker images.
+
+## Image updates from Trusty
+
+Xenial includes the following changes and improvements:
+
+### Third party apt-repositories removed
+
+While third party apt-repositories are used during the Xenial image provisioning, they are all removed from the Xenial build image. This has two benefits; a) reduced risk of unrelated interference and b) faster apt-get updates.
+
+To specify a third party apt-repository, you can [add the source with the apt addon](/user/installing-dependencies/#adding-apt-sources) and specify the packages. For example:
+
+```yaml
+dist: xenial
+addons:
+  apt:
+    sources:
+      - ppa:chris-lea/redis-server
+    packages:
+    - redis-tools
+    - redis-server
+```
+{: data-file=".travis.yml"}
+
+### Third party packages available in Xenial
+
+The following table outlines the third party packages available in Xenial, along with their original source.
+
+| package       | source                                                                      |
+|:--------------|:----------------------------------------------------------------------------|
+| redis         | `ppa:chris-lea/redis-server`                                                |
+| haskell       | `ppa:hvr/ghc`                                                               |
+| mongodb       | `"deb http://repo.mongodb.org/apt/ubuntu xenial/mogodb-org/4.0 multiverse"` |
+| couchdb       | `ppa:couchdb/stable`                                                        |
+| docker        | `"deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"` |
+| git-lfs       | `"deb https://packagecloud.io/github/git-lfs/ubuntu/ xenial main"`          |
+| git-ppa       | `ppa:git-core/ppa`                                                          |
+| google_chrome | `"deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main"`     |
+| pollinate     | `ppa:pollinate/ppa`                                                         |
+
+### Services disabled by default
+
+On Trusty, some services that you might or might not use, were running by
+default. On Xenial, to optimize the use of the virtual machine resources, they come disabled by default.
+
+You can start them by specifying the services that you require. For example:
+
+```yaml
+services:
+  - mysql
+  - redis
+```
+{: data-file=".travis.yml"}
+
+## Environment common to all Xenial images
+
+In total, three different Xenial images are available for use. The selection
```

Comment:
These paragraphs are well confusing, and we don't mention `steveonnie` anywyere else. I'd b tempted to either delete all of these paragraphs up to the common environment, or copy some of text from Ubuntu about Minimal and Generic.

WDYT?